api/lib/schema.js

```diff
@@ -72,6 +72,8 @@ const typeDefs = gql`
     id: String
     voter: NetworkAccount
     option: String
+    amount: Coin # Substrate only
+    time: String # Substrate only
```

Comment:
we don't get time in the end, am I wrong?

Reply:
ping

Reply:
Yeah, I think this is already in anyway. Was part of #4887

Reply:
yes, it is. closing this one now